aiida/storage/psql_dos/migrations/utils/legacy_workflows.py

```diff
@@ -41,9 +41,9 @@ def export_workflow_data(connection, profile):
     DbWorkflowData = table('db_dbworkflowdata')
     DbWorkflowStep = table('db_dbworkflowstep')
 
-    count_workflow = connection.execute(select(func.count()).select_from(DbWorkflow)).scalar()
-    count_workflow_data = connection.execute(select(func.count()).select_from(DbWorkflowData)).scalar()
-    count_workflow_step = connection.execute(select(func.count()).select_from(DbWorkflowStep)).scalar()
+    count_workflow = connection.execute(select(func.count()).select_from(DbWorkflow)).scalar()  # pylint: disable=not-callable
```

Comment:
Yes, they are simply due to the upgrade to sqlalchemy v2 but there is not much we can do I think. Or it would be quite time consuming to figure out in any case and is not worth doing at this point in time